src/Okta.Idx.Sdk/IdxClient.cs

```diff
@@ -205,30 +205,38 @@ private string GenerateCodeChallenge(string codeVerifier, out string codeChallen
         /// </summary>
         /// <param name="state">Optional value to use as the state argument when initiating the authentication flow. This is used to provide contextual information to survive redirects.</param>
         /// <param name="cancellationToken">The cancellation token. Optional.</param>
-        /// /// <param name="activationToken">The activation token. Optional.</param>
+        /// <param name="activationToken">The activation token. Optional.</param>
+        /// <param name="recoveryToken">The recovery token. Optional.</param>
         /// <returns>The IDX context.</returns>
-        internal async Task<IIdxContext> InteractAsync(string state = null, CancellationToken cancellationToken = default, string activationToken = null)
+        internal async Task<IIdxContext> InteractAsync(string state = null, CancellationToken cancellationToken = default, string activationToken = null, string recoveryToken = null)

```

Comment:
Where does one receive a recovery token to begin this kind of interaction?

Reply:
@bryanapellanes-okta it's returned as a part or the response of `ForgotPasswordGenerateOneTimeTokenAsync` Management SDK call.

Reply:
Is the thought that those using the Management Sdk will also use the Idx Sdk in conjunction?  Or, is the management Sdk using the idx Sdk somehow?

Reply:
It seems like it's only supported by ManagementSDK now. The Jira description says `recoveryToken is minted from user api`
In [this PR for okta-idx-android](https://github.com/okta/okta-idx-android/pull/115/files#diff-6eb2d0320631edbd9cfa761242d530295593d44f27756983c38aabdd8b76e54bR56) it's also taken from ManagementSDK